aten/src/ATen/native/Normalization.cpp

```diff
@@ -372,7 +372,7 @@ std::tuple<Tensor, Tensor, Tensor, int64_t> _batch_norm_impl_index(
   if (use_cudnn && eps >= detail::getCUDAHooks().batchnormMinEpsilonCuDNN()) {
     return std::tuple_cat(
              at::cudnn_batch_norm(
-               input.contiguous(), weight.contiguous(),
+               input.contiguous(input.suggest_memory_format()), weight.contiguous(),
```

Comment:
Are we sure that weight should be contiguous?

Reply:
cudnn batchnorm requires weight to be in 1xCx1x1(x1) for our SPATIAL mode:
https://docs.nvidia.com/deeplearning/sdk/cudnn-developer-guide/index.html#cudnnDeriveBNTensorDescriptor

I checked the kernel we are using nhwc ones so we should be good.